wordpress-mobile/WordPress-Android
# Issue/10630 material my site

Fixes #10630

This PR moves My Site screen and navbar to the material theme and adds Dark Theme support. There are some code changes for cosmetic reasons.

Design changes are reviewed before the code review is requested.

While refactoring is not in the scope of this PR, since I touched those files, I cleaned up/optimized some of the redundant code as suggested by AS, so do not be alarmed by the size of the PR. I suggest turning on the "Hide whitespace changes" option during the review.

There is a huge amount of PRs in this feature that covers all the screens within the app, so I'm requesting reviews from a wide variety of people.

![comparison_mysite](https://user-images.githubusercontent.com/728822/72587942-9c3c9300-38ab-11ea-967d-d085993bf6c6.png)


To test:
Navigate to My Site fragment and confirm it's visual and functional integrity in both Light and Dark themes. The theme can be changed from App Settings.

PR submission checklist:
